src/bundle.c

```diff
@@ -800,6 +804,7 @@ gboolean check_bundle(const gchar *bundlename, RaucBundle **bundle, gboolean ver
 					R_SIGNATURE_ERROR,
 					R_SIGNATURE_ERROR_X509_NEW,
 					"failed to allocate new X509 store");
+			res = FALSE;
```

Comment:
`res` is `TRUE` here as otherwise we wouldn't have reached this line. The same is true for the following hunk - good catch!

Reply:
Thanks. Updated the PR, please take a look again.